tests/run_test.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash
-# tests/run_test_suite.sh.  Generated from run_test_suite.sh.in by configure.
 
-# run_test_suite
+# run_test
 #
-# Copyright (C) 2013,2016 by Swiss Library for the Blind, Visually Impaired and Print Disabled 
```

Comment:
Are the years 2013, 2016 not relevant anymore?

Now that I'm on this subject, I just read in the [GNU Coding Standards](https://www.gnu.org/prep/maintain/maintain.html#Copyright-Notices) that:

> You can use a range (‘2008-2010’) instead of listing individual years (‘2008, 2009, 2010’) if and only if: 1) every year in the range, inclusive, really is a “copyrightable” year that would be listed individually; and 2) you make an explicit statement in a README file about this usage. 

I didn't know this. In the past I have sometimes used a range when not all the years in the range were really "copyrightable". I did this mainly in cases in which the copyright notices in a table file had not been kept up to date, and I went looking it up in the Git history. However it is tricky because it is not always clear 1) if the committer is also the author and 2) whether a particular change is a big enough change to warrant a copyright notice. As I result I would sometimes put a whole range so that I wouldn't have to worry about details.

Reply:
Ah, sorry was to zealous with rming stuff. Will fix.